sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/models/AmqpMessageBodyType.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.models;
+
+/**
+ * Represents all valid {@link AmqpMessageBodyType} for an AMQP Message. Current SDK only support
+ * {@link AmqpMessageBodyType#DATA DATA} AMQP data type. Track this <a href="https://github.com/Azure/azure-sdk-for-java/issues/17614" target="_blank">issue</a>
+ * to find out support for other AMQP types.
```

Comment:
Same here - javadoc doesn't have to include future release issues. We can just state that currently only DATA type is supported.

Reply:
Jonathan specifically asked me to create an issue and include which user can subscribe to,